packages/terra-clinical-item-display/src/terra-dev-site/doc/example/TruncatedText.jsx

```diff
@@ -2,7 +2,8 @@ import React, { useState } from 'react';
 import ItemDisplay from 'terra-clinical-item-display';
 import ShowHide from 'terra-show-hide';
 
-const sampleText = 'Mr. James is currently receiving outpatient treatment. He has been diagnosed with an Axis I diagnosis of Psychosis NOS, ruled out Schizoaffective Disorder and Post Traumatic Stress Disorder, and is being treated with Haldol 5mg and Cogentin 1mg.';
+const previewText = 'Mr. James is currently receiving outpatient treatment. He has been diagnosed with an Axis I diagnosis of Psychosis NOS, ruled out Schizoaffective Disorder and Post Traumatic Stress Disorder, and is being treated with Haldol 5mg and Cogentin 1mg.';
+const sampleText = 'Mr. James is currently receiving outpatient treatment. He has been diagnosed with an Axis I diagnosis of Psychosis NOS, ruled out Schizoaffective Disorder and Post Traumatic Stress Disorder, and is being treated with Haldol 5mg and Cogentin 1mg. His treatment may have potential side effects such as dizziness and headaches. His symptoms have been manageable, if any side effects or concerns come up, consult with the doctor immediately.';
```

Comment:
To make the ShowHide accessible per recent component update (to make the assistive technologies announce the previously cut (hidden) portion instead of reading the whole text from the beginning (which would make the preview part be read twice)), the preview portion and the rest of the text needs to be separated and passes as props to `ShowHideFocuser` component.
```suggestion
const sampleText = 'His treatment may have potential side effects such as dizziness and headaches. His symptoms have been manageable, if any side effects or concerns come up, consult with the doctor immediately.';
```

Reply:
Per discussion with @trandrew1023 all suggested by me changes to ShowHide example were added in https://github.com/cerner/terra-clinical/pull/908/commits/8774eceee376e0a91f19e62db3c6212aa0fcb47f